components/scream/tests/uncoupled/p3/p3_stand_alone_output.yaml

```diff
@@ -0,0 +1,12 @@
+%YAML 1.1
+---
+Casename: p3_stand_alone_output
+Averaging Type: Instant
+Grid: Physics
+Max Snapshots Per File: 1
+Fields: [T_mid, T_prev_micro_step, qv, qc, qr, qv_prev_micro_step, eff_radius_qc, eff_radius_qi,
```

Comment:
nitpicky, and not necessary, but this might be a good place to use the `-` example for making long lists of fields.

Reply:
Done